src/navigation/vertical-navigation.js

```diff
@@ -0,0 +1,156 @@
+const component = {};
+
+/**
+  * @ngdoc directive
+  * @name bdVerticalNavigation
+  * @module buildium.angular-elements.navigation
+  * @restrict E
+  * 
+  * @description
+  * 
+  * A vertical navigation element suitable for sidebar navs
+  *
+  * @param {Array} [navigationItems] - The collection of navigation items that should be added
+  * - <a class="label type-hint type-hint-string">string</a> `title` - <strong>Name to display for this navigation item</strong>
```

Comment:
really minor but I think it's best if we include href params even in examples, just so that people don't copy pasta invalid HTML

Reply:
I have hrefs in the first example here, it is optional though